widget/toolbar.go

```diff
@@ -17,13 +17,16 @@ type ToolbarItem interface {
 type ToolbarAction struct {
 	Icon        fyne.Resource
 	OnActivated func() `json:"-"`
+	disable     bool
```

Comment:
Wording this as "disable" sounds more like you want someone to disable it. I think "disabled" is a better name as the field specifies if the component is disabled or not.